<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerSettingsDialog</class>
 <widget class="QDialog" name="ViewerSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/16x16/actionViewDisplaySettings.png</normaloff>:/icons/16x16/actionViewDisplaySettings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QComboBox" name="viewComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="projectionLabel">
         <property name="text">
          <string>Projection</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="projectionComboBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="qualitySlider">
         <property name="maximum">
          <number>4</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="qualityLabel">
         <property name="text">
          <string>Quality</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="viewLabel">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="backgroundColorLabel">
         <property name="text">
          <string>Background color</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="backgroundColorPushButton"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="labelsTab">
      <attribute name="title">
       <string>Labels</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="labelsOnAtomsGroupBox">
         <property name="title">
          <string>On atoms</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="2">
           <widget class="QSpinBox" name="labelsOnAtomsFontSizeSpinBox">
            <property name="minimum">
             <number>6</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFontComboBox" name="labelsOnAtomsFontComboBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelsOnAtomsTypeLabel">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelsOnAtomsFontLabel">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="labelsOnAtomsTypeComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../rsrc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
